templates/app/views/spree/products/_product_header.html.erb

```diff
@@ -0,0 +1,24 @@
+<header class="product-header">
+  <h1 class="product-header__title" itemprop="name">
+    <%= product.name %>
+  </h1>
+
+  <% if product.price_for_options(current_pricing_options)&.money and !product.price.nil? %>
```

Comment:
This price nil check is in Starter Frontend but not in the demo. The following spec fails if it is removed. However, the test failure only occurs in CI. I wasn't able to figure out why the failure doesn't appear locally.